collectd/files/write_graphite.conf

```diff
@@ -10,16 +10,18 @@
 LoadPlugin write_graphite
 
 <Plugin write_graphite>
- <Carbon>
-   Host "{{ collectd_settings.plugins.write_graphite.host }}"
-   Port "{{ collectd_settings.plugins.write_graphite.port }}"
-   Prefix "{{ collectd_settings.plugins.write_graphite.prefix }}."
-   Postfix "{{ collectd_settings.plugins.write_graphite.postfix }}"
-   Protocol "{{ collectd_settings.plugins.write_graphite.protocol }}"
-   EscapeCharacter "{{ collectd_settings.plugins.write_graphite.escapecharacter }}"
-   LogSendErrors {{ collectd_settings.plugins.write_graphite.logsenderrors|lower }}
-   SeparateInstances {{ collectd_settings.plugins.write_graphite.separateinstances|lower }}
-   StoreRates {{ collectd_settings.plugins.write_graphite.storerates|lower }}
-   AlwaysAppendDS {{ collectd_settings.plugins.write_graphite.alwaysappendds|lower }}
- </Carbon>
+{%- for node_name, graphite_node in collectd_settings.plugins.write_graphite.nodes.iteritems() %}
+ <Node "{{ node_name }}">
```

Comment:
You need to version key this or just stick with Carbon for now. Some versions (notably the version of collectd in debian stable) don't understand <Node> (vs <Carbon>). FWIW, the code in collectd master still accepts Carbon.